src/crystal/system/unix/file.cr

```diff
@@ -108,9 +108,9 @@ module Crystal::System::File
     raise ::File::Error.from_errno("Error changing owner", file: path) if ret == -1
   end
 
-  def self.fchown(path, fd, uid : Int, gid : Int)
+  private def system_chown(uid : Int, gid : Int)
```

Comment:
I'd keep the `path` parameter, since it's kinda odd to use an ivar here.

Reply:
Not any more odd than `fd`.
But `#system_chmod` passes a `path` argument.
But `#system_truncate` expects the `#path` method.

Let's harmonize to expect the `#path` method just like we expect the `#fd` method.